static/chart.js

```diff
@@ -89,9 +93,9 @@ const drawChart = async () => {
             ).attr("transform",
                 "translate(" + margin.left + ", " + margin.top + ")")
             .attr("id", "id"+index)
-            .attr("class", "timeSeries cluster-All");
+            .attr("class", "timeSeries " + elt[2] + " cluster-All");
```

Comment:
Can we use a better variable name than "elt"? It's OK to be a little more verbose if your meaning is clearer. If you know what the element of an array represents, I think it's clearer to avoid generic words like element.

Reply:
Thank you! I realized arrays can be unpacked in JS.